network/crl_test.go

```diff
@@ -0,0 +1,350 @@
+/*
+Copyright 2020, Cossack Labs Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+)
+
+func getTestCACert() []byte {
+	// tests/ssl/ca/ca.crt
+	return []byte(`
```

Comment:
Those certificates might be a bit painful to update once they expire (or rather, when something inside them becomes deprecated).

I guess it would make more sense to just read them from the files directly. However, it's a bit tricky to do in all the contexts the tests are executed (IDEs, CI runners...) I remember @Lagovas mentioning once that Go is getting some magic macro that will expand into file's content at compile time. It would be nice to leave a TODO for future refactoring, I think.